ddtrace/internal/_encoding.pyx

```diff
@@ -580,10 +581,11 @@ cdef class MsgpackEncoderV03(MsgpackEncoderBase):
             ret = pack_number(&self.pk, span.trace_id)
             if ret != 0: return ret
 
-            ret = pack_bytes(&self.pk, <char *> b"parent_id", 9)
-            if ret != 0: return ret
-            ret = pack_number(&self.pk, span.parent_id)
-            if ret != 0: return ret
+            if has_parent_id:
```

Comment:
Do you have updated performance numbers with this change?

Reply:
oh, I thought that was from when we were doing `if span.parent_id is not None` twice.

thanks